File: Easy/2026-07-30-3014-Minimum-Number-of-Pushes-to-Type-Word-I/solution.java
class Solution {
    public int minimumPushes(String word) {
        int n = word.length();
        int pushes = 0;
        
        // Each key can hold at most 3 letters, so we calculate the number of pushes
        for (int i = 0; i < n; i++) {
            pushes += (i / 3) + 1;
        }
        
        return pushes;
    }
}

File: Easy/2026-07-30-3014-Minimum-Number-of-Pushes-to-Type-Word-I/solution.py
class Solution:
    def minimumPushes(self, word: str) -> int:
        n = len(word)
        # Each letter can be assigned to a key with a maximum of 3 letters per key
        pushes = 0
        for i in range(n):
            pushes += (i // 3) + 1
        return pushes